packages/components/src/Payments/PaymentsAmount.tsx

```diff
@@ -56,13 +59,21 @@ export class PaymentsAmount extends React.Component<PaymentsAmountProps, Payment
         </RadioInput>
         <PaymentAmountUserInput>
           {this.state.showInput ? (
-            <CurrencyInput icon={<>$</>} name="CurrencyInput" onChange={this.handleInput} />
+            <CurrencyInput icon={<>$</>} name="CurrencyInput" onChange={this.handleAmountInput} />
           ) : (
             <PaymentsShowInputBtn onClick={this.handleShowInput}>
               <EnterCustomAmountText />
             </PaymentsShowInputBtn>
           )}
         </PaymentAmountUserInput>
+        <PaymentAmountUserComment>
```

Comment:
should put comments behind a feature flag, since we don't want to turn them on until there's some kind of moderating ability for channel admins